assignments/assignment1.md

```diff
@@ -1 +1,73 @@
-TODO
\ No newline at end of file
+# Trading Card Games
+
+## Objective
+The aim of this assignment is to familiarize you with interacting with smart contracts deployed on a blockchain, specifically the MultiversX blockchain. Through this assignment, you will learn how to interact with a deployed smart contract, manage, and exchange Card Games (Non-Fungible Tokens - NFTs).
+
+## Learning Outcomes
+By the end of this assignment, students will be able to:
+
+- Understand and execute transactions on the MultiversX blockchain;
+- Interact with a deployed smart contract by calling methods using the Python SDK (mxpy tool);
+- Use NFT trading functionality provided by our smart contract;
+- Query on-chain data from our smart contract and interpret results;
+- Gain a better understanding of blockchain concepts while performing a transaction:
+  - Private key
+  - Gas limit
+  - Proxy
+  - Nonce
+
+## Assignment Tasks
+
+### 1. Setup Environment:
+- Set up a MultiversX wallet to be used for testnet interactions;
```

Comment:
devnet is more stable. I recommend using devnet.
Also, maybe add a note that if they already have a wallet, they don't need to create one.  This may be used to track their progress over the semester.
You may recommend to create a new one to track their progress particularly for this assignment. Your call